firmware/defmt-rtt/README.md

```diff
@@ -0,0 +1,42 @@
+# `defmt-rtt`
+
+> Transmit [`defmt`] log messages over the RTT (Real-Time Transfer) protocol
+
+[`defmt`]: https://github.com/knurling-rs/defmt
+
+`defmt` ("de format", short for "deferred formatting") is a highly efficient logging framework that targets resource-constrained devices, like microcontrollers.
+
+The fastest way to get started with `defmt` is to use our [app-template] to set up a new Cortex-M embedded project.
+
+[app-template]: https://github.com/knurling-rs/app-template
+
+For more details about the framework check the book at https://defmt.ferrous-systems.com
```

Comment:
```suggestion
For more details about the framework check the book at https://defmt.ferrous-systems.com.
```

Reply:
adding a period there may break the URL (not sure myself) so I'll add this with a space after the URL